LibraBFT/Impl/Consensus/Liveness/RoundState.agda

```diff
@@ -44,5 +44,5 @@ insertVoteM vote verifier = do
 -- TODO-1: Implement this.
 -- > recordVote v = rsVoteSent ∙= just v
 recordVote : Vote → LBFT Unit
-recordVote v = pure unit
+recordVote v = rsVoteSent-rm ∙= just v
```

Comment:
I have started using the naming convention we discussed (but perhaps did not settle on?) regarding type-classed Haskell lenses.

Reply:
Cool, I think that's an improvement.  However, I note that there are "unsolved meta" warnings on the setters.  I guess we need to finish defining `α-EC` before we can fix this?

Reply:
My guess is also that α-EC being unfinished is the cause of those unsolved meta-variables.